internal/cmd/operator-sdk/generate/internal/genutil.go

```diff
@@ -34,6 +34,7 @@ import (
 
 // ValidateVersion returns an error if version is not a strict semantic version.
 func ValidateVersion(version string) error {
+	version = strings.TrimPrefix(version, "v")
 	v, err := semver.Parse(version)
```

Comment:
```suggestion
	v, err := semver.ParseTolerant(version)
```
`semver.ParseTolerant` (https://github.com/blang/semver/blob/af3461a9cbcf1f3f5889d21b83f5ef63880c33a8/v4/semver.go#L240) would sanitize the string for us.

Reply:
I reverted this change. The test was failing with this, complaining the version do not match.